tests/cleanup/remove_old_vms_openstack.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# Script removes virtual machines older than HOURS_LIMIT (24 hours by default) from OpenStack
+
+# TODO
+# * remove old images
```

Comment:
The name pattern is `Composer-$UUID-Automated-Import`.

Also need to remove volumes b/c sometimes they may not be removed automatically after removing the VM.